premerge/premerge_resources/main.tf

```diff
@@ -47,6 +47,48 @@ resource "kubernetes_namespace" "llvm_premerge_windows_2022_runners" {
   }
 }
 
+resource "kubernetes_namespace" "llvm_premerge_linux_buildbot" {
```

Comment:
Same comment about names.

Reply:
Copy the buildbot-named things comment here too?

Reply:
Done.